conjure-runtime/src/service/http_error.rs

```diff
@@ -118,14 +119,24 @@ where
             }
             _ => {
                 let (parts, body) = response.into_parts();
-
-                let body = match Limited::new(body, 10 * 1024).collect().await {
-                    Ok(body) => body.to_bytes(),
-                    Err(e) => {
-                        info!("error reading response body", error: Error::internal(e));
-                        Bytes::new()
+                let stream = body.into_data_stream();
+                pin_mut!(stream);
```

Comment:
nit: you can use std::pin to make this `let stream = pin!(body.into_data_stream())`.